### PR TITLE
feat: add error handler to all API operations

### PR DESCRIPTION
**Pull request by DiegoHBustamante**
This PR presents the following changes.

- Try-catch block to all API requests
- Redirection to error handling page in case of a failed stream

Captura de pantalla 2024-12-10 a la(s) 12 36 53 p  m

Captura de pantalla 2024-12-05 a la(s) 10 47 35 a  m

Captura de pantalla 2024-12-11 a la(s) 3 49 24 p  m

### CODE REVIEW

**Review by mvarlic — APPROVED** [review]
LGTM!

File: src/helpers/resultError.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface ResultError
+  {   
+      errorMessage: string;
+  }
```
Comment (mastudillot) [inline src/helpers/resultError.ts:4]: add en line

Comment (DiegoHBustamante) [reply src/helpers/resultError.ts]: line added

**Review by mastudillot — APPROVED** [review]
LGTM

**Pull request merged**